main.nf

```diff
@@ -4,7 +4,7 @@
     nf-core/cutandrun
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
     Github : https://github.com/nf-core/cutandrun
-    Website: https://nf-co.re/cutandrun
+Website: https://nf-co.re/cutandrun
```

Comment:
```suggestion
    Website: https://nf-co.re/cutandrun
```
I think it's due to TEMPLATE sync, we fixed that in dev, but tools hasn't been re-released yet

Reply:
should be tabbed out in next release which I am about to apply